bqplot/scales.py

```diff
@@ -346,12 +346,25 @@ class LinearScale(Scale):
         two or more scales have the same rtype and dtype.
         default_value is 2 because for the same range and domain types,
         LinearScale should take precedence.
+    stabilized: bool (default: False)
+        if set to False, the domain of the scale is tied to the data range
+        if set to True, the domain of the scale is udpated only when
+        the data range is beyond certain thresholds, given by the attributes
+        mid_range and min_range.
+    mid_range: float (default: 0.8)
+        Proportion of the range that is spanned initially.
+    min_range: float (default: 0.6)
+        Minimum proportion of the range that should be spanned by the data.
+        If the data span falls beneath that level, the scale is reset.
```

Comment:
Can you expand more on the meaning of `mid_range` and `min_range`, whether they are only used if stabilized is `True`.

mid_range seems to be used to divide a quantity. Would it make sense for the inverse to be passed instead?